### ISSUE TITLE
Provide source XML location as comment next to term

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/stepmod-utils#30]
There are a number of terms/definitions that need fixing due to misspellings and typos, and we need to fix them not only in ISO 10303-2 but also in the source STEPmod repository. In order to make those fixes in the source repo we want to know the location of the file to make the changes.

e.g.
```
# From: data/modules/assembly_feature_definition/module.xml
=== assembmly feature relationship

a relationship between a pair of the assembly features of the assembled components

[.source]
<<ISO-TS_10303-1102_2005,clause=3.2>>
```

**Comment by TRThurman** [comment]
We also need to record the rcs revision number
cvs status 

should be 

"Up-to-date"

And record the value of

"Working revision"

> On Jul 7, 2020, at 1:23 AM, Ronald Tse <notifications@github.com> wrote:
> 
> 
> There are a number of terms/definitions that need fixing due to misspellings and typos, and we need to fix them not only in ISO 10303-2 but also in the source STEPmod repository. In order to make those fixes in the source repo we want to know the location of the file to make the changes.
> 
> e.g.
> 
> # From: data/modules/assembly_feature_definition/module.xml
> === assembmly feature relationship
> 
> a relationship between a pair of the assembly features of the assembled components
> 
> [.source]
> <<ISO-TS_10303-1102_2005,clause=3.2>>
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> 

Thomas Thurman
Principal Electrical Engineer, Rockwell Collins (Retired)

Confidentiality Notice: This e-mail (including attachments) is covered by the Electronic Communications Privacy Act, 18 U.S.C. 2510-2521, is confidential and may be legally privileged.  If you are not the intended recipient, you are hereby notified that any retention, dissemination, distribution, or copying of this communication is strictly prohibited.  Please reply to the sender that you have received the message in error, then delete/destroy any and all copies of the original message.
E-mail transmissions cannot be guaranteed to be secure or error-free as information could be intercepted, corrupted, lost, destroyed, arrive late,  incomplete, or 
contain viruses. Therefore, the recipient should check this email and any attachments for the presence of viruses. The sender accepts no liability for any damage caused by any virus transmitted by this email.

**Comment by TRThurman** [comment]
Please explain how cvs command gets called to provide the requested cvs status data.
e.g., how to get this result:
# From: data/modules/assembly_feature_definition/module.xml v 1.3

**Comment by ronaldtse** [comment]
@TRThurman I'm doing this from our Git copy of the CVS repo. Will update it to use the CVS repo and provide that information.

**Comment by ronaldtse** [comment]
@TRThurman this is now implemented in #31.
